Makefile

```diff
@@ -1,6 +1,9 @@
 # using the chart name and version from chart's metadata
 CHART_NAME ?= $(shell awk '/^name:/ { print $$2 }' Chart.yaml)
-CHART_VESION ?= $(shell awk '/^version:/ { print $$2 }' Chart.yaml)
+CHART_VERSION ?= $(shell awk '/^version:/ { print $$2 }' Chart.yaml)
```

Comment:
Ouch! Thank you 🙏

Reply:
😛 I spend a few mintues figuring out why it would create `task-git-` as a folder 😛